airbyte-integrations/bases/standard-destination-test/src/main/java/io/airbyte/integrations/standardtest/destination/DestinationAcceptanceTest.java

```diff
@@ -972,7 +971,11 @@ protected void runSyncAndVerifyStateOutput(JsonNode config,
     final AirbyteDestination destination = getDestination();
 
     destination.start(destinationConfig, jobRoot);
-    messages.forEach(message -> Exceptions.toRuntime(() -> destination.accept(message)));
+    try {
+      messages.forEach(message -> Exceptions.toRuntime(() -> destination.accept(message)));
+    } catch (Exception e) {
```

Comment:
Are you sure that nothing is supposed to handle this Exception higher in the call stack?

Reply:
Thank you. I didn't notice it and didn't delete it after debugging.